### DISCUSSION TITLE
Question about note titles

### DISCUSSION THREAD

**Discussion by kavanaram** [argenos/zotero-mdnotes#72] [Q&A]
Thanks for this great tool :)

I have a question about the name of the md file - my zotero files follow the naming convention "<Lastname> (<year>) <Title before special Characters>" eg. Lastname (3025) Effects of Teleportation on Gut Microbiome

Is there any way to have mdnotes retain that format for the notes it creates? I'm guessing it's not possible, but since I'm not well-versed in this, I'm asking in case there's something I might have missed

**Comment by argenos** [comment]
Hi @Estelm4! Right now there is no way for mdnotes to track when you change the file names for the `mdnotes` file; it's a rather hard-coded assumption based on the [naming convention](https://argentinaos.com/zotero-mdnotes/docs/configuration/#file-naming-convention).

Are you (re-)naming those files manually? Maybe the [`Standalone menu`](https://argentinaos.com/zotero-mdnotes/docs/templates/#standalone-note-template) would work for you? (It needs to be enabled in the settings).
